letsencrypt-auto-source/letsencrypt-auto.template

```diff
@@ -110,7 +125,7 @@ su_sudo() {
 
 SUDO_ENV=""
 export CERTBOT_AUTO="$0"
-if [ -n "${LE_AUTO_SUDO:+x}" ]; then
```

Comment:
Sorry for my previous comment. A test like `if [ -n "${LE_AUTO_SUDO+x}" ]; then` does in fact work in busybox's shell.